src/Game/UI/Gumps/TopBarGump.cs

```diff
@@ -109,9 +109,18 @@ private TopBarGump() : base(0, 0)
 
             string[] texts =
             {
-                ResGumps.Map, ResGumps.Paperdoll, ResGumps.Inventory, ResGumps.Journal, ResGumps.Chat, ResGumps.Help,
-                ResGumps.WorldMap, ResGumps.Info, ResGumps.Debug, ResGumps.NetStats, ResGumps.UOStore,
-                ResGumps.GlobalChat
+                cliloc.GetString(3000430,ResGumps.Map), 
+                cliloc.GetString(3000133,ResGumps.Paperdoll), 
+                cliloc.GetString(3000431,ResGumps.Inventory), 
+                cliloc.GetString(3000129,ResGumps.Journal), 
+                cliloc.GetString(3000131,ResGumps.Chat), 
+                cliloc.GetString(3000134,ResGumps.Help),     
+                ResGumps.WorldMap), 
+                ResGumps.Info), 
+                ResGumps.Debug), 
+                ResGumps.NetStats), 
+                ResGumps.UOStore),
```

Comment:
Can you add the translation for uostore & global chat too?

Here the clilocs thanks
cliloc.GetString(1158008, ResGumps.UOStore),
cliloc.GetString(1158390, ResGumps.GlobalChat)

Reply:
Sure, I didn't found them at Clilocs